src/utils.py

```diff
@@ -34,6 +34,10 @@ def normalized_cut_2d(edge_index: torch.Tensor, pos: torch.Tensor):
     return normalized_cut(edge_index, edge_attr, num_nodes=pos.size(0))
 
 
-def calculate_accuracy(outputs: torch.Tensor, targets: torch.Tensor):
-    pred = outputs.max(1)[1]
-    return pred.eq(targets).sum().item()
+def calculate_accuracy(outputs: torch.Tensor, targets: torch.Tensor) -> float:
+    """
+    calculates the accuracy between the argmax of the last dimension of outputs
+    and the targets
+    """
+    pred = outputs.argmax(-1)
+    return pred.view_as(targets).eq(targets).sum().item()
```

Comment:
I think I found the bug: the returned should have been divided by the size of the batch::  return `pred.view_as(targets).eq(targets).sum().item() / targets.numel()`

Reply:
ooh ty! i'll try it out!